Godeps/Godeps.json

```diff
@@ -688,13 +653,23 @@
 			"Rev": "5da22b92e9485c15a6b284d52cc54d6580864c99"
 		},
 		{
+			"ImportPath": "gopkg.in/inf.v0",
+			"Rev": "3887ee99ecf07df5b447e9b00d9c0b2adaa9f3e4"
+		},
+		{
 			"ImportPath": "k8s.io/kubernetes/pkg/api/resource",
-			"Comment": "v0.12.0-270-g53ec66caf4e9",
-			"Rev": "53ec66caf4e952a1384ec93b9f0cde37616e4caf"
+			"Comment": "v1.3.0-alpha.4-205-g9625926",
```

Comment:
k8s is so fluent that i'm not so surprised.
in general - i have to be +1 for this.